### PR TITLE
Task Calculator

### PR DESCRIPTION
**Pull request by AlekseyGrimm**
https://github.com/rolling-scopes-school/tasks/blob/master/tasks/stage-0/projects.md#task-1-calculator

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<html lang="en"><head>
+	<meta charset="UTF-8">
+	<title>Calculator</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+	<link rel="stylesheet" href="style.css">
+	<link rel="shortcut icon" href="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRJ6QT1nxjEwatD664uyBM4Vgy9TNaVNmo-8w&usqp=CAU">
```
Comment (y-solovey) [inline index.html]: please update href to your image

File: script.js [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+		if (this.operation != null) {
+			this.previousOperandTextElement.innerText =
+				`${this.getDisplayNumber(this.previousOperand)} ${this.operation}`;
+		} else {
+			this.previousOperandTextElement.innerText = ''
+		}
+	}
+}
+
+const numToFixed = x => ~(x + '').indexOf('.') ? (x + '').split('.')[1].length : 0;
```
Comment (y-solovey) [inline script.js]: where it is used?

File: script.js [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+		this.empty = '';
+		this.equal = false;
+		this.squar = false;
+
+
+	}
+
+	delete() {
+		this.currentOperand = this.currentOperand.toString().slice(0, -1); 
+		// delete one numder
```
Comment (y-solovey) [inline script.js]: please move comments before the code

Comment (AlekseyGrimm) [reply script.js]: done

File: index.html (hunk 1 above)
Comment (AlekseyGrimm) [reply index.html]: delete href

File: script.js (hunk 2 above)
Comment (AlekseyGrimm) [reply script.js]: I deleted it, this is code not important

**Review by y-solovey — APPROVED** [review]

**Pull request closed without merging**